projekt_do_pracy_na_labach/fiszki_backend/Dockerfile

```diff
@@ -1,2 +1,55 @@
 FROM python:3.10
 
+WORKDIR /fiszki_backend
+
+COPY . /fiszki_backend
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+ENV "DATABASE_URL"="mongodb://mongoContainer:27017"
+
+EXPOSE 8000
+
+CMD ["python", "main.py"]
+
+# 3.5
+# docker build -t fiszki_backend .
+# docker run -d --name fiszki_backend_cont --rm -p 8000:8000 fiszki_backend
+#
+# zeby zadzialalo stworzenie nowego kontenera z networkiem:
+# sudo docker rm -f <id konterera stworzonego wczesniej>
+#
+# 4.0
+# docker network create appnetwork
+# docker run -d --name fiszki_backend_network --rm -p 8000:8000 --network=appnetwork fiszki_backend
+# docker run -d --name mongo_network -e DATABASE_URL=mongodb://mongoContainer:27017 --network=appnetwork mongo
+# docker network inspect appnetwork
+#
+# 4.5
+# docker run -d --name fiszki_backend --rm -p 800:800 -v vol1:/fiszki_backend fiszki_backend
+# docker run -d --name mongo --rm -p 80:800 -v vol1:/fiszki_backend mongo
```

Comment:
Musisz podać miejsce gdzie mongo przechowuje dane. Podpowiedź do tego znajdziesz [tutaj](https://hub.docker.com/_/mongo)

Reply:
Hmm, dobra trochę napisałem to jak debil (sorry jak coś XD)
W sensie masz dobrze zrobiony wolumen do mongo od strony technicznej (faktycznie utworzy Ci wolumen do folderu `fiszki_backend` w kontenerze mongo) a nie praktycznej (w tym folderze nie są przechowywane dane przez mongodb). Chodziło mi w tej podpowiedzi by sobie "podpatrzeć" lokację, gdzie twórcy obrazu montują ten wolumen